pkg/destroy/aws/aws.go

```diff
@@ -1308,26 +1308,27 @@ func deleteEC2SubnetsByVPC(ctx context.Context, client *ec2.EC2, vpc string, fai
 				},
 			},
 		},
+		func(results *ec2.DescribeSubnetsOutput, lastPage bool) bool {
+			for _, subnet := range results.Subnets {
+				err := deleteEC2Subnet(ctx, client, *subnet.SubnetId, logger.WithField("subnet", *subnet.SubnetId))
+				if err != nil {
+					err = errors.Wrapf(err, "deleting EC2 subnet %s", *subnet.SubnetId)
+					if lastError != nil {
```

Comment:
This could print the same error multiple times. Is this intended or should this only print new errors?

Reply:
We discussed this on Slack and we don't think the error will be printed multiple times. Also, this was the existent behaviour already.